occupancy-quickstart/src/appSettings.cs

```diff
@@ -8,10 +8,12 @@
 namespace Microsoft.Azure.DigitalTwins.Samples
 {
     public class AppSettings {
+        private static string DigitalTwinsAppId = "0b07f429-9f4b-4714-9392-cc5e8e80c8b0";
```

Comment:
@fernandoviton - not sure if I've configured this wrong following the change, I've modified both this as well as the `appSettings.json` with my AAD ID. I'm getting a:

```
We don't recognize this code, please enter the code again.
```

Reply:
Hmm.  I would be curious what type of config could lead to that error.  Can you send me your appSetting.json and/pr appSetting.dev.json file offline?